cmd/helmitTypes.go

```diff
@@ -21,3 +23,26 @@ type HelmitResponse struct {
 	Error    bool            `json:"error"`
 	Replicas []HelmitReplica `json:"replicas"`
 }
+
+//ShipmentStatus represents the deployed status of a shipment
+type ShipmentStatus struct {
+	Namespace string `json:"namespace"`
+	Version   string `json:"version"`
+	Status    struct {
+		Phase      string `json:"phase"`
+		Containers []struct {
+			Image    string `json:"image"`
+			Ready    bool   `json:"ready"`
+			Restarts int    `json:"restarts"`
+			State    struct {
+				Running struct {
```

Comment:
Ok, I added support for unhealthy states (example below).  I'm hoping `CrashLoopBackOff` along with the `LAST STATE` field helps tell the story of what's happening to a container.  Also provides some context for the `RESTARTS` field.  Let me know what you think.  Also are there any other states that we should be accounting for?

```
SHIPMENT:      mss-poc-thingproxy2   
ENVIRONMENT:   dev                   
STATUS:        Running               
CONTAINERS:    1                     
REPLICAS:      2

IMAGE                                                         STATUS    STARTED            RESTARTS   LAST STATE                 
registry.services.dmtio.net/mss-poc-thingproxy:0.0.13-rc.33   waiting   CrashLoopBackOff   17         terminated 2 minutes ago   
registry.services.dmtio.net/mss-poc-thingproxy:0.0.13-rc.33   waiting   CrashLoopBackOff   17         terminated 5 minutes ago  
```